chain/client/src/client_actor.rs

```diff
@@ -1132,6 +1134,26 @@ impl ClientActor {
         let now = Utc::now();
 
         let timer = metrics::CHECK_TRIGGERS_TIME.start_timer();
+
+        self.flat_storage_creation_next_attempt = self.run_timer(
```

Comment:
I think the flat_storage_creation calls should not be part of check_triggers, instead, you can schedule the timer in `ClientActor::started` like `catchup` and `start_sync`. The reason is, the events in `check_triggers` will be scheduled at a higher priority than all actix messages (see the comments at the beginning of `check_triggers` and see https://github.com/near/nearcore/blob/master/chain/client/src/client_actor.rs#L281). So we need to careful at what we put in `check_triggers` as this work can block other important jobs such as processing new incoming blocks. In the case of flat storage creation, I think it should happen at the lower priority than processing incoming messages, since it is not time sensitive.